src/components/hint/index.js

```diff
@@ -343,19 +353,18 @@ class Hint extends BaseComponent {
   }
 
   renderHint() {
-    const {position, message, messageStyle, icon, iconStyle, borderRadius, color, testID} = this.getThemeProps();
-    const shownUp = position === HINT_POSITIONS.TOP;
+    const {message, messageStyle, icon, iconStyle, borderRadius, color, testID} = this.getThemeProps();
 
     if (this.showHint) {
       return (
-        <AnimatableView
-          animation={shownUp ? AnimatableManager.animations.hintAppearUp : AnimatableManager.animations.hintAppearDown}
-          duration={200}
+        <View
+          animated
```

Comment:
how do you control the type of animation to apply on the View?

Reply:
What do you mean?

Reply:
With Animatable you can load the animations, like 'hintApearUp'. How do you do it with the 'animated' prop of View. What animation does the Hint will have now?

Reply:
Got it. getHintAnimatedStyle

Reply:
The default one. 
Users won't be able to configure it now. 
I don't thing people were aware of that and I think it's pretty redundant..
Of course if someone will request we can reconsider it